docs/pages/project/contributing/contributing-ui.md

```diff
@@ -9,50 +9,125 @@ category: contributing
 ---
 
 ### <a name="contributing-ui">UI Contribution Flow</a>
+
 Meshery is written in `Go` (Golang) and leverages Go Modules. UI is built on React and Next.js. To make building and packaging easier a `Makefile` is included in the main repository folder.
 
+### Architecture
+
+The following is a list of top-level frameworks, libraries, design system used in Meshery UI.
+
+- [NextJS](https://nextjs.org/) - Server and router
+- [ReactJS](https://reactjs.org/) - User Interface library
+- [Material UI](https://material-ui.com/) - Design System
+- [BillboardJS](https://naver.github.io/billboard.js/) - Charting library, used for exposing Grafana and Prometheus-collected metrics
+
+- [MeshMap only] [CytoscapeJS](https://js.cytoscape.org/) - a visualization tool for canvas-based, visual topology (networks)
+
+#### Meshery Server APIs
+
+Go [here](https://docs.meshery.io/extensibility/api#rest) for the docs.
+
+**REST API**
+
+- Meshery provides a REST API available through the default port of `9081/tcp`.
+- List of [endpoints](https://docs.meshery.io/reference/rest-apis) (spreadsheet) a simple, static list of REST API endpoints with short description of their purpose.
+- Swagger / Open API.
+- Collection of sets of REST API docs that Meshery server exposes to clients (like the Meshery UI).
+
+**GraphQL API**
+
+- Meshery provides a GraphQl API available through the default port of `9081/tcp`.
+- [Relay](https://relay.dev) is the client used.
+
+### Design
+
+#### Wireframing / Mockups
+
+- Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
+
+  > You need to ask for access in [Slack](http://slack.layer5.io/)
```

Comment:
@leecalcote How about adding a link to the Community Member form along with the slack? So that if a person needs to access it, he has to fill the form at first 👀